search-sentiments-map/routes/trends.js

```diff
@@ -23,10 +23,15 @@ const datastore = new Datastore();
 
 const TRENDS_DATA_KIND = 'TrendsEntry';
 const STALE_DATA_THRESHOLD_7_DAYS_MS = 7 * 24 * 60 * 60000;
+// Constant for getting popularity data from 8 days ago.
+const DATA_FROM_8_DAYS_AGO_MS = 8 * 24 * 60 * 60000; 
```

Comment:
This is a threshold/start time. Please rename appropriately.

We name constants in a way that indicates their use.
The current name is more of a comment-style, if that makes sense.

Reply:
renamed!